### DISCUSSION TITLE
Tree Supports

### DISCUSSION THREAD

**Discussion by timmcfadden** [OrcaSlicer/OrcaSlicer#234] [Ideas]
Any chance you can add the tree supports like PrusaSlicer 2.6 is doing?  They are based off this https://www.reddit.com/r/3Dprinting/comments/p7n4x4/i_created_a_new_tree_support_implementation_for/  Might be easier for prusa slicer to implement them and just use their code.

**Comment by lilyclark0124** [comment]
Tree supports in PrusaSlicer 2.6 are really elegant because they minimize contact points, reduce [filament](https://kingroon.com/collections/3d-printer-filament?sca_ref=9443968.VmbECjyAZgWd) usage, and make support removal much easier.
